gradio/examples.py

```diff
@@ -153,9 +153,9 @@ def __init__(
             self.cache_interface_examples()
 
         def load_example(example_id):
-            processed_example = self.processed_examples[example_id]
+            processed_input = self.processed_examples[example_id]
             if cache_examples:
-                processed_example += self.load_from_cache(example_id)
```

Comment:
The cached output was being appended again and again to the input, causing the bug